ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -233,5 +233,15 @@ SENSOR_ZA_CO2M_PMM_HCHO:
   - CO2M
   - PMM
   uses:
-  - zone_air_formaldehyde_concentration_sensor  
-  
\ No newline at end of file
+  - zone_air_formaldehyde_concentration_sensor
+
+SENSOR_PMM2X_CO2M13X:
```

Comment:
I do not understand why these would be modeled together. Instead, model each individual zone CO2 sensor and then connect them to the floor in building config. If you are doing this because you have a device that is proxying this data for you (e.g. a sensor gateway) then use the `GATEWAYS/PASSTHROUGH` entity type, so you dont have to define a bespoke, one-off entity type. Then, using links in Building Config, you can link the gateway to the individual sensor models you create.

Reply:
Actually this all 13 CO2 sensors , 2 PM2.5 Sensors are inside a single area served by a single AHU.

Reply:
your description for `CO2M13X` designates these as 13 separate zones. are these really all placed in one control zone?